test/MigrationTest.sol

```diff
@@ -9,38 +9,35 @@ import {EncodeLib} from "./helpers/libraries/EncodeLib.sol";
 import {IERC20} from
     "../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 
-contract MorphoTokenEthereumMigrationTest is BaseTest {
-    address internal constant BUNDLER_ADDRESS = 0x4095F064B8d3c3548A3bebfd0Bbfd04750E30077;
-    address internal constant LEGACY_MORPHO = 0x9994E35Db50125E0DF82e4c2dde62496CE330999;
+interface RolesAuthority {
+    function setUserRole(address user, uint8 role, bool enabled) external;
+}
 
-    IMulticall internal bundler;
-    IERC20 internal legacyMorpho;
+contract MigrationTest is BaseTest {
+    IMulticall internal bundler = IMulticall(0x4095F064B8d3c3548A3bebfd0Bbfd04750E30077);
+    IERC20 internal legacyMorpho = IERC20(0x9994E35Db50125E0DF82e4c2dde62496CE330999);
 
     uint256 internal forkId;
 
     bytes[] internal bundle;
 
     function setUp() public virtual override {
+        super.setUp();
+
```

Comment:
It's just a bit unclear to me what is the expected behavior if `createSelectFork` is done after some transactions like it is done in `super.setUp()` here. For example, in which block would the contracts considered to be created ?